src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Rectangle extends Figure {
+    private static final String type = "rectangle";
```

Comment:
The 'type' field should be an instance variable, not a class variable since it's not a constant value. Also, consider using a constructor to set the type if it's meant to be immutable.